tasks.py

```diff
@@ -1350,3 +1350,17 @@ def docs_server(c, address='localhost:8080', compile_schema=False):
         c.run('python docs/extract_schema.py docs/schema.yml')
 
     c.run(f'mkdocs serve -a {address} -f docs/mkdocs.yml')
+
+
+@task
+def clear_generated(c):
+    """Clear generated files from `inv update`."""
+    # pyc/pyo files
+    c.run('find . -name "*.pyc" -exec rm -f {} +')
```

Comment:
Can we ensure that `c.run` is executed from the local dir, no matter where the `invoke` command is called from? Otherwise this may present an issue

Reply:
I have added a general helper to ensure this